src/main/java/cinema/controllers/CinemaHallController.java

```diff
@@ -0,0 +1,36 @@
+package cinema.controllers;
+
+import cinema.mappers.CinemaHallMapper;
+import cinema.model.dto.CinemaHallRequestDto;
+import cinema.model.dto.CinemaHallResponseDto;
+import cinema.service.CinemaHallService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinemahalls")
+public class CinemaHallController {
+    @Autowired
+    CinemaHallService cinemaHallService;
+    @Autowired
+    private CinemaHallMapper cinemaHallMapper;
+
+    @PostMapping("/add")
```

Comment:
```suggestion
    @PostMapping
```
![image](https://user-images.githubusercontent.com/50640794/84520185-d5065a00-acdb-11ea-96bd-f5b569aa8a41.png)